File: src/Z.Core/System.Int32/System.Drawing.ColorTranslator/Int32.FromWin32.cs
// Description: C# Extension Methods | Enhance the .NET Framework and .NET Core with over 1000 extension methods.
// Website & Documentation: https://csharp-extension.com/
// Issues: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License (MIT): https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: https://zzzprojects.com/
// Copyright © ZZZ Projects Inc. All rights reserved.
using System;
#if !NETSTANDARD
using System.Drawing;
#endif
public static partial class Extensions
{
#if !NETSTANDARD
    /// <summary>
    ///     Translates a Windows color value to a GDI+  structure.
    /// </summary>
    /// <param name="win32Color">The Windows color to translate.</param>
    /// <returns>The  structure that represents the translated Windows color.</returns>
    public static Color FromWin32(this Int32 win32Color)
    {
        return ColorTranslator.FromWin32(win32Color);
    }
#endif
}

File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Int32/System.Drawing.ColorTranslator/Int32.FromWin32.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library

Imports System.Drawing

Public Module Extensions_286

	''' <summary>
	'''     Translates a Windows color value to a GDI+  structure.
	''' </summary>
	''' <param name="win32Color">The Windows color to translate.</param>
	''' <returns>The  structure that represents the translated Windows color.</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function FromWin32(win32Color As Int32) As Color
		Return ColorTranslator.FromWin32(win32Color)
	End Function
End Module


